content/2017.04.getting-started-with-elm.rst

```diff
@@ -112,9 +126,9 @@ with Elm files.
 Starting your first project
 ===========================
 
-No need for boilerplate here, you can just start by running
-``elm-package install`` and approve the default core packages that
-will be installed:
+No need for a boilerplate here, you can just start by running
```

Comment:
@glasserc said we should remove the a in front of boilerplate because boilerplate is uncountable.